src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4056,27 +4057,35 @@ namespace winrt::TerminalApp::implementation
 
             TitlebarBrush(acrylicBrush);
             bgColor = backgroundColor;
+
+            setBG = true;
         }
-        else if (theme.TabRow() && theme.TabRow().Background())
+        else if (theme.TabRow())
         {
-            const auto tabRowBg = theme.TabRow().Background();
-            const auto terminalBrush = [this]() -> Media::Brush {
-                if (const auto& control{ _GetActiveControl() })
-                {
-                    return control.BackgroundBrush();
-                }
-                else if (auto settingsTab = _GetFocusedTab().try_as<TerminalApp::SettingsTab>())
-                {
-                    return settingsTab.Content().try_as<Controls::Page>().Background();
-                }
-                return nullptr;
-            }();
+            if (const auto tabRowBg{ _activated ? theme.TabRow().Background() :
+                                                  theme.TabRow().UnfocusedBackground() })
+            {
+                const auto terminalBrush = [this]() -> Media::Brush {
+                    if (const auto& control{ _GetActiveControl() })
+                    {
+                        return control.BackgroundBrush();
+                    }
+                    else if (auto settingsTab = _GetFocusedTab().try_as<TerminalApp::SettingsTab>())
+                    {
+                        return settingsTab.Content().try_as<Controls::Page>().Background();
+                    }
+                    return nullptr;
```

Comment:
Why not write this?
```cpp
Media::Brush terminalBrush{ nullptr };
if (...)
{
    terminalBrush = ...;
}
else if (...)
{
    terminalBrush = ...;
}
```

That's the way we've written these things in a lot of other places.

Reply:
.... Because I clearly wrote this with very little to no sleep 😅 yea wtf was I thinking.